docs(rspamd): Add guidance for setting WebUI password (#4023)
docker-mailserver/docker-mailserver: docs/content/config/security/rspamd.md

![Rspamd Web Interface](https://rspamd.com/img/webui.png)

> To make use of the web interface you need to setup setup a password otherwise you won't be able to log in.
> To do so you must
> [set the `password` option for the controller worker to a password hash generated by `rspamadm pw`](#with-the-help-of-a-custom-file).
> 
> ??? example "Configuration example"
>     Below is an example of how the configuration should look like.
> 
>     ```bash
>     set-option-for-controller password "yourHashedPassword"
>     ```
> 

Reviewer: ```suggestion
To use the web interface you will need to configure a password, otherwise you won't be able to log in.

??? example "Set a custom password"

    Add this line to [your rspamd `custom-commands.conf` config](#with-the-help-of-a-custom-file) which sets the `password` option of the _controller worker_:

    ```
    set-option-for-controller password "your hashed password here"
    ```

    The password hash can be generated via the `rspamadm pw` command:

    ```bash
    docker exec -it dms rspamadm pw
    ```

```

![Rspamd Web Interface](https://rspamd.com/img/webui.png)

> To use the web interface you will need to configure a password, otherwise you won't be able to log in.

Reviewer: ```suggestion
To use the web interface you will need to configure a password, [otherwise you won't be able to log in][rspamd-docs::web-ui::password].
[rspamd-docs::web-ui::password]: https://www.rspamd.com/doc/tutorials/quickstart.html#setting-the-controller-password
```

??? example "Set a custom password"

    Add this line to [your rspamd `custom-commands.conf` config](#with-the-help-of-a-custom-file) which sets the `password` option of the _controller worker_:

    ```
    set-option-for-controller password "your hashed password here"
    ```

    The password hash can be generated via the `rspamadm pw` command:

    ```bash

>     docker exec -it dms rspamadm pw

Author: From searching docs I get that the commands shown in the docs are either

`docker exec -it <CONTAINER_NAME>` or `docker exec -it CONTAINER_NAME`

and `docker exec -it mailserver`

I'd stick with `docker exec -it <CONTAINER_NAME>` since it's the first command example that you come across in the docs as it's in the [`Usage` section](https://docker-mailserver.github.io/docker-mailserver/latest/usage/#get-up-and-running).

>     docker exec -it dms rspamadm pw

Author: From searching docs I get that the commands shown in the docs are either

`docker exec -it <CONTAINER_NAME>` or `docker exec -it CONTAINER_NAME`

and `docker exec -it mailserver`

I'd stick with `docker exec -it <CONTAINER_NAME>` since it's the first command example that you come across in the docs as it's in the [`Usage` section](https://docker-mailserver.github.io/docker-mailserver/latest/usage/#get-up-and-running).

>     docker exec -it dms rspamadm pw

Reviewer: Good spotting, thanks! 😁 

```suggestion
    docker exec -it <CONTAINER_NAME> rspamadm pw
```
